src/client/idv.client.js

```diff
@@ -92,6 +92,18 @@ class IDVClient {
   getSupportedDocuments() {
     return this.idvService.getSupportedDocuments();
   }
+
+  /**
+   * Fetches the configuration for the given sessionID.
+   *
+   * @param {string} sessionId
+   * @return {SessionConfigurationResponse}
+   *
+   * @returns {Promise}
```

Comment:
I used a bit the `examples/idv` to try it out, works fine!
But I realised that we are not really doing a good job with the JSDoc when it comes to promise.

Could you please update the `idv.client.js` and `idv.service.js` JSDoc comments so:
- there is only 1 `@return`
- use `return` or `returns` (which ever is the one the most used in the project)
- replace any `@returns {Promise}` by `@returns {Promise<TheClassItResolvesWith>}`

I had a look about the reject/error, but there is nothing available in JSDoc it seems... so, we'll get these when on Typescript later.